docs/sources/whatsnew/whats-new-in-v9-3.md

```diff
@@ -0,0 +1,244 @@
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-photos-9-3-0.png" max-width="750px" caption="Geomap panel photos layer" >}}
+
+## Canvas panel
+
+Available in **beta** in all editions of Grafana
+
+Canvas is a new panel that combines the power of Grafana with the flexibility of custom elements. Canvas visualizations are extensible form-built panels that allow you to explicitly place elements within static and dynamic layouts. This empowers you to design custom visualizations and overlay data in ways that aren’t possible with standard Grafana panels, all within Grafana’s UI. If you’ve used popular UI and web design tools, then designing Canvas panels will feel very familiar.
+
+In Grafana v9.3 We have added icon value mapping support to the Canvas panel. This enables you to dynamically set which icon to display based on your data. To learn more about the Canvas panel, see the [documentation]({{< relref "../panels-visualizations/visualizations/canvas" >}}).
+
+{{< video-embed src="/static/img/docs/canvas-panel/canvas-icon-value-mapping-support-9-3-0.mp4" max-width="750px" caption="Canvas panel icon value mapping support" >}}
+
+## Public dashboards improvements
+
+### Manage all of your public dashboards in one place
+
+Available in **experimental** in Grafana Open Source, Enterprise, and Cloud Advanced
+
+You can use Public Dashboards to make a given dashboard available to anyone on the internet with no need to sign in. In Grafana v9.3 we have introduced a new screen where you can manage all of your public dashboards. From here, you can view is a list of all of the public dashboards in your Grafana instance, navigate to the underlying dashboard, see if it is enabled, link out to the public version of the dashboard, or update the public dashboard's configuration. You can see a public dashboard's configuration if you have view access to the dashboard itself, and you can edit its configuration if you have the Admin or Server Admin role, or the "Public Dashboard writer" role if you are using RBAC in Grafana Enterprise or Cloud Advanced.
+
+To check out this new screen and configure your public dashboards, navigate to Dashboards → Public Dashboards. 
+
+[image public-dashboard-audit-table.png]
+
+### Choose to display annotations in public dashboards
+
+Available in **experimental** in Grafana Open Source, Enterprise, and Cloud Advanced
+
+Annotations are now supported in public dashboards, with the exception of query annotations. They are turned off by default, but can be turned on in your public dashboard settings.
+
+Note that since Public Dashboards is an experimental feature, you need to enable it in Grafana using the `publicDashboards` [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}), or open a support ticket requesting public dashboards if you are a Cloud Advanced customer.
```

Comment:
```suggestion
Note that because Public Dashboards is an experimental feature, you need to enable it in Grafana using the `publicDashboards` [feature toggle]({{< relref "../setup-grafana/configure-grafana/#feature_toggles" >}}), or open a support ticket requesting public dashboards if you are a Cloud Advanced customer.
```